### ISSUE TITLE
Broken resolution when starting Onikakushi in fullscreen mode

### ISSUE THREAD

**Issue by enumag** [07th-mod/higurashi-assembly#12]
Platform: Windows 10 x64
Using 2 monitors, main one is WQHD (that's where Higurashi opens) the other is FullHD.

Orian had the same problem, I think he is on windows too but I'm not sure about his monitors.

![fullscreen](https://user-images.githubusercontent.com/539462/47867849-02520d00-de03-11e8-8da7-e4f5db9dd1f7.png)

@tellowkrinkle I'm guessing you broke it somehow when trying to fix the other resolution-related bugs.

**Comment by orian34** [comment]
I have only one monitor, still does it.

**Comment by tellowkrinkle** [comment]
[Assembly-CSharp_Onikakushi.dll.zip](https://github.com/07th-mod/higurashi-assembly/files/2539747/Assembly-CSharp.dll.zip)
Could you run with this and tell me what it says in the log about `Using resolution <stuff>`?

**Comment by enumag** [comment]
Here is the entire log. There is a good number of various errors.

[output_log.txt](https://github.com/07th-mod/higurashi-assembly/files/2539811/output_log.txt)

**Comment by enumag** [comment]
These are what I found particularly interesting:

```
Mismatched serialization in the builtin class 'Texture2D'. (Read 524360 bytes but expected 524372 bytes)
```

(this one is there multiple times)

and

```
The file 'C:/Program Files (x86)/Steam/steamapps/common/Higurashi When They Cry/HigurashiEp01_Data/mainData' is corrupted! Remove it and launch unity again!
[Position out of bounds!]
```

**Comment by tellowkrinkle** [comment]
Could you test with this one?
[Assembly-CSharp_Onikakushi.dll.zip](https://github.com/07th-mod/higurashi-assembly/files/2540072/Assembly-CSharp.dll.zip)

**Comment by enumag** [comment]
Yeah, that fixed it. Here is the log in case you need it: 
[output_log.txt](https://github.com/07th-mod/higurashi-assembly/files/2540082/output_log.txt)

It still contains the errors I posted above - I'm guessing they are unrelated and not really important then.

@orian34 Can you test that DLL too to be sure?

**Comment by tellowkrinkle** [comment]
[Assembly-CSharp_Onikakushi.dll.zip](https://github.com/07th-mod/higurashi-assembly/files/2540108/Assembly-CSharp.dll.zip)
Actually, can you test this one as well?  The other one was a bit of an awkward workaround, and I just realized the main cause of the issue so if this one works as well I'd rather use it.

**Comment by orian34** [comment]
Seems like it works.

**Comment by enumag** [comment]
@tellowkrinkle Yeah, the second one works fine too!

**Comment by tellowkrinkle** [comment]
Fixed in 2e9c421dd3ad5bd89f01991b12dd421e691c2d87
